src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < actions.length; i++) {
```
'For of' loop would be more readable